package.json

```diff
@@ -5,12 +5,15 @@
   "scripts": {
     "format": "prettier --config .prettierrc '**/*.ts' --write",
```

Comment:
It seems the single quotes don't work on Windows, when I replace it with `**/*.ts` instead of `'**/*.ts'` it works.